combat-trainer.lic

```diff
@@ -1425,6 +1425,13 @@ class SpellProcess
     true
   end
 
+  def cast_runestone(data, game_state)
+    prepare_to_cast_runestone(data, @settings)
+    return unless DRCI.in_hands?('runestone')
```

Comment:
I know I originally suggested passing in just 'runestone' as the argument, but I wonder if we should pass in the actual runestone name per the config? I see it's used at line 1432, so this would be more consistent? Also handles if for some odd reason a runestone exists that _didn't_ use the runestone noun?
```suggestion
    return unless DRCI.in_hands?(data['runestone_name'])
```

Reply:
Changed and testing.